platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulers.java

```diff
@@ -252,6 +254,13 @@ public static PlatformSchedulers create(
                         .withUnhandledTaskCapacity(config.issDetectorUnhandledCapacity())
                         .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
                         .build()
+                        .cast(),
+                model.schedulerBuilder("hashLogger")
+                        .withType(TaskSchedulerType.SEQUENTIAL)
+                        .withOnRamp(hashingObjectCounter)
+                        .withExternalBackPressure(true)
+                        .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
```

Comment:
We are trying too keep the threading model the same as much as possible, even when switching to the new framework initially. At some point in the near future, we will look at optimizing it. For now, everything that was a `QueueThread` should become a `SEQUENTIAL_THREAD` with whatever capacity was set (100 in this case). Also, I don't think it needs any external back-pressure.

Reply:
is the capacity something that should be made configurable?

Reply:
made it configurable anyway.